Static/TableViewController.swift

```diff
@@ -27,62 +20,40 @@ public class TableViewController: UIViewController {
 
     // MARK: - Initialization
 
-    public init(style: UITableViewStyle) {
-        tableView = UITableView(frame: .zero, style: style)
-        super.init(nibName: nil, bundle: nil)
-        dataSource.tableView = tableView
+    public override init(style: UITableViewStyle) {
+        super.init(style: style)
+        initialize()
     }
 
     public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: NSBundle?) {
-        tableView = UITableView(frame: .zero, style: .Plain)
         super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
-        dataSource.tableView = tableView
+        initialize()
     }
 
     public required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
-
-    public convenience init() {
-        self.init(style: .Plain)
+        super.init(coder: aDecoder)
+        initialize()
     }
 
 
     // MARK: - UIViewController
 
-    public override func loadView() {
-        tableView.autoresizingMask = [.FlexibleWidth, .FlexibleHeight]
-        view = tableView
-    }
-
     public override func viewWillAppear(animated: Bool) {
         super.viewWillAppear(animated)
         performInitialLoad()
-        clearSelectionsIfNecessary()
-    }
-
-    public override func viewDidAppear(animated: Bool) {
-        super.viewDidAppear(animated)
-        tableView.flashScrollIndicators()
-    }
-
-    public override func setEditing(editing: Bool, animated: Bool) {
-        super.setEditing(editing, animated: animated)
-        tableView.setEditing(editing, animated: animated)
     }
 
 
     // MARK: - Private
 
+    private func initialize() {
+        dataSource.tableView = tableView
+    }
+
     private var initialLoadOnceToken = dispatch_once_t()
```

Comment:
I think this stuff can go too, since it's specific to this reimplementation of UITableViewController: https://github.com/venmo/Static/pull/30#discussion_r36920942